services/slam/builtin/cartographer_int_test.go

```diff
@@ -141,20 +141,22 @@ func integrationTestHelperCartographer(t *testing.T, mode slam.Mode) {
 
 	testCartographerPositionAndMap(t, svc)
 
+	// Sleep to ensure cartographer saves at least one map
+	time.Sleep(time.Second * time.Duration(*attrCfg.MapRateSec))
```

Comment:
LGTM, may consider adding the same sleep to orbslam's offline test. Orbslam takes longer to run so it may not be necessary but would be safer

Reply:
Let's do that when we have to